src/test/nodestore/Database_test.cpp

```diff
@@ -27,7 +27,10 @@
 #include <test/nodestore/TestBase.h>
 #include <test/unit_test/SuiteJournal.h>
 
+#include <memory>
```

Comment:
I don't think this `#include` is necessary.

Reply:
removed